tests/unit/stubs/stubs/mbed/BLEKit.h

```diff
@@ -0,0 +1,23 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ble/BLE.h"
+#include "ble/Gap.h"
+
+namespace leka {
+
+extern FunctionPointerWithContext<BLE::InitializationCompleteCallbackContext *>
+	spy_ble_initialization_complete_callback;
+extern bool spy_ble_init_calls;
+
+extern bool spy_ble_has_initialized;
+
+extern FunctionPointerWithContext<ble::BLE::OnEventsToProcessCallbackContext *> spy_ble_on_events_to_process_callback;
+
+void spy_ble_init_reset_calls();
```

Comment:
si possible, je ferai une method qui reset tout

```suggestion
void spy_ble_reset();
```